README.md

```diff
@@ -40,58 +40,31 @@ To generate a client library using the OpenAPI spec, use the provided script:
 $ ./bin/generate_client <language>
 ```
 
-The script with generate a Python client in the absence of a `<language>` argument.  
-A full list of supported languages can be found in [OpenAPI Generator documentation](https://github.com/OpenAPITools/openapi-generator#overview).
+The script will generate a Python client in the absence of a `<language>` argument.  
+A full list of supported languages can be found in 
```

Comment:
Unrelated to this PR: Should we re-evaluate this being the default behavior? It's based on the assumption that if someone leaves out an argument, they would rather have a Python client be accidentally created over doing nothing at all. I think it would be a less costly UX change to exit with an error, requiring that the user pass in an argument.

Reply:
I'm in favor of changing this. I was thinking about updating the script to be able to specify output directory, I can hit both of these at once. I'll file some issues.